base_app/src/app/dataset-tables/dataset-table.html

```diff
@@ -11,11 +11,14 @@
                                     <th colspan="2" class="tableTitle" align="center" [uid]="_dataset.u_id" >
                                         <div [className]="this._hidden ? 'sectionClosed':'sectionOpened'"
                                             [title]="fixNL(_dataset.desc)"
-                                            (click)="toggleTableDisplay()"
-                                        >&nbsp;</div>
-                                        <span *ngIf="!_dataset.url">{{ getTitle() }}</span
-                                        >&nbsp;<a *ngIf="_dataset.url" target="_blank"  href={{_dataset.url}}
-                                                  title={{fixNL(_dataset.tooltip)}}>{{ getTitle() }}</a>
+                                            (click)="toggleTableDisplay()">&nbsp;</div>
+
+                                        <span *ngIf="!_dataset.url">{{ getTitle() }}</span>
+                                            &nbsp;
```

Comment:
I'm pretty sure, but not positive, that using the spaces _outside_ of `< ... >` versus _inside_ means that you will have a "regular / breakable space", rather than an unbreakable space.  Thus,   
    `<foo> &nbsp; </foo>` may render unwanted breaks, whereas  
    `<foo>&nbsp;</foo>` will not.
    
That's the reason I coded the `>` to break across a line - to prevent unwanted breaks.

Reply:
I think your right. I fixed the formatting in my last commit